spanner/client.go

```diff
@@ -116,6 +116,8 @@ type ClientConfig struct {
 	// override the default values.
 	CallOptions *vkit.CallOptions
 
+	DatabaseRole string
```

Comment:
Add a comment.

"DatabaseRole specifies the role to be assumed for all operations on the database by this client."